src/main/webapp/WEB-INF/tags/student/profile/uploadPhotoModal.tag

```diff
@@ -57,20 +57,41 @@
                             </c:when>
                             <c:otherwise>
                                 <div class="profile-pic-edit">
-                                    <div class="profile-pic-edit-container">
-                                        <img id="editableProfilePicture" src="${modal.pictureUrl}">
+                                    <div class="alert alert-info">
+                                        Zoom, pan and rotate to crop your image to the desired area.
                                     </div>
+                                    <button id="profilePicEditPanUp" type="button" class="btn btn-primary">
+                                        <span class="glyphicon glyphicon-arrow-up"></span>
+                                    </button>
+                                    <div class="profile-pic-edit-container row">
+                                        <div class="col-xs-2">
+                                            <button id="profilePicEditPanLeft" type="button" class="btn btn-primary">
+                                                <span class="glyphicon glyphicon-arrow-left"></span>
+                                            </button>
+                                        </div>
+                                        <div class="col-xs-8">
+                                            <img id="editableProfilePicture" src="${modal.pictureUrl}">
+                                        </div>
+                                        <div class="col-xs-2">
+                                            <button id="profilePicEditPanRight" type="button" class="btn btn-primary">
+                                                <span class="glyphicon glyphicon-arrow-right"></span>
+                                            </button>
+                                        </div>
+                                    </div>
+                                    <button id="profilePicEditPanDown" type="button" class="btn btn-primary">
+                                        <span class="glyphicon glyphicon-arrow-down"></span>
+                                    </button>
```

Comment:
God mode is still not working perfectly with this.
God mode changed the blob-key argument to
${regkey.enc}:L2dzL3RlYW1tYXRlcy1hc2hyYXkuYXBwc3BvdC5jb20vY2hhcmxpZS50bW1zLmFzaHJheQ

But on the staging server, it expects ${regkey.enc}. Somehow, the key is not being stripped out correctly.